src/unicode/capi.h

```diff
@@ -0,0 +1,76 @@
+/**
+ * This file is part of the "libunicode" project
+ *   Copyright (c) 2021 Christian Parpart <christian@parpart.family>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
```

Comment:
Wouldn't be a classical include guard the safer bet here? Imho the pragma support is still relatively new, and since this lib is meant to get used in a variety of circumstances (from static linkage to some exotic language binding), it might create unwanted frictions with older compilers.

Reply:
The historical reasons for avoiding `#pragma once` has been that NFS mount points potentially created inode conflicts and GCC used inodes as quick comperators. That however is at least 20 years ago. I am not sure if there was another reason, but at least NFS is not a problem anymore.

It is however (I think) more common to see the other form of header guards in C-style code though.

Reply:
True, its not new at all, but in C terms new means it is not at least 40ys old :smile_cat:
Last time I used the C compiler in VS it was still on C89. And that was like 3 ys ago, haha.